cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -682,55 +761,78 @@ public void checkCustomCaSecret(CertificateAuthority ca, Secret certSecret, Secr
             }
         }
 
-        Future<ReconciliationState> kafkaUpgrade() {
+        private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSs) {
+
+            // Get the current version of the cluster
+            KafkaVersion currentVersion = versions.version(Annotations.annotations(kafkaSs).get(ANNO_STRIMZI_IO_KAFKA_VERSION));
+
+            log.debug("Does SS {} Kafka version need to be changed?", kafkaSs.getMetadata().getName());
```

Comment:
In general - this message seems to make no sense. Doesn't add any information over the message right below. We should remove it.